### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by joaquinbozzalla**
## Summary ##

**Trello card**: https://trello.com/c/4HJdkNBc

First unit tests of the app.

### CODE REVIEW

File: BooksTests/BookRepositoryTest.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import XCTest
+@testable import WBooks
+
+class BookRepositoryTest: XCTestCase {
+
+    var sut: BookRepositoryType!
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        sut = BookRepository()
```
Comment (gmazzei) [inline BooksTests/BookRepositoryTest.swift:17]: Looks good! Just one comment about repositories:
Tests should be repeatable and isolated if possible. This means that some entities that interact with external APIs, like repositories, are not meant to be tested. For example, if one day the API breaks down, you would see test failing even though your app code is totally fine 👀  
It's just one thing to keep in mind, but the test looks good! 😉

File: BooksTests/BookRepositoryTest.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        let onSuccess = { (books: [Book]) in
+            promise.fulfill()
+        }
+        let onError = { (error: Error) in
+            XCTAssertNotNil(error, "Error in books request: \(error)")
+        }
+        
+        sut.fetchBooks(onSuccess: onSuccess, onError: onError)
+        wait(for: [promise], timeout: 20)
+    }
```
Comment (gmazzei) [inline BooksTests/BookRepositoryTest.swift:37]: Nice use of expectations 👏

File: WBooks/Models/Book.swift [hunk 3]
```diff
@@ -11,7 +11,7 @@ class UnidentifiableBook: Codable {
     let genre: String
     let year: String
     let image: String
-    var status : BookStatus
+    var status : BookStatus = .unavailable
```
Comment (gmazzei) [inline WBooks/Models/Book.swift]: Is the default value necessary?

Comment (gmazzei) [reply WBooks/Models/Book.swift]: Remove the space between `status` and the `:`

**Review by gmazzei — APPROVED** [review]
LGTM! 🥇

File: BooksTests/BookRepositoryTest.swift (hunk 1 above)
Comment (joaquinbozzalla) [reply BooksTests/BookRepositoryTest.swift:17]: Great, thanks!

**Pull request merged**